news.md

```diff
@@ -66,32 +66,32 @@ please see [TUF Augmentation Proposals
 Riyaz Faizullabhoy from Docker gave a
 [talk](https://lcccna2016.sched.org/event/7JWU/when-the-going-gets-tough-get-tuf-going-riyaz-faizullabhoy-docker)
 on TUF and Notary at LinuxCon North America.  Slides of his talk are available
-here:
-https://events.linuxfoundation.org/events/linuxcon-north-america/program/slides
+[here](https://events.linuxfoundation.org/events/linuxcon-north-america/program/slides).
+
+**March 18, 2016**
+
+> 
+Trishank Kuppusamy presents "Diplomat: Using Delegations to Protect Community
+Repositories" at [NSDI 2016](https://www.usenix.org/conference/nsdi16).
```

Comment:
Should we link to the page that contains the presentation slides, audio recording, paper, etc.?

https://www.usenix.org/node/194973